components/treelist/state.md

```diff
@@ -185,20 +188,26 @@ The following example shows one way you can store the TreeList state - through a
         JsInterop.InvokeVoidAsync("window.location.reload");
     }
 
-    private void SetState()
+    private async void SetState()
```

Comment:
We should never use `async void`, as it doesn't work correctly sometimes.

```suggestion
    private async Task SetState()
```

Reply:
We are currently recommending to use `async void`  `OnStateChanged` handler - https://docs.telerik.com/blazor-ui/components/treelist/state#events.

The rest I have updated.